src/module.c

```diff
@@ -2305,7 +2305,7 @@ mstime_t RM_GetExpire(RedisModuleKey *key) {
  * The function returns REDISMODULE_OK on success or REDISMODULE_ERR if
  * the key was not open for writing or is an empty key. */
 int RM_SetExpire(RedisModuleKey *key, mstime_t expire) {
-    if (!(key->mode & REDISMODULE_WRITE) || key->value == NULL)
+    if (!(key->mode & REDISMODULE_WRITE) || key->value == NULL || (expire < 0 && expire != REDISMODULE_NO_EXPIRE))
```

Comment:
Too long. Should split it into new line?

Reply:
i don't personally mind long lines up to a reasonable limit.
any hard limit (like 80) would be wrong, IMHO is should be decided per case to make the code readable.
i.e. i hate it when a line is split in odd places just to comply with a limit, which makes it unreadable.

in this case, splitting the `if` line would mean the `return` would be indented the same as the `if` line and then we better add a `{` on a line of it's own.
i.e. we'll be adding 3 lines.
i personally think the current code is ok, but i don't mind splitting it if it bothers anyone.

p.s. if this was error handling code that's repeated many times, and breaking it to many lines would make it hard to read the rest of / actual code and logic of the function, i would argue to keep it one long line.